app/assets/javascripts/Components/submission_table.jsx

```diff
@@ -163,7 +163,7 @@ class RawSubmissionTable extends React.Component {
     {
       Header: I18n.t('activerecord.attributes.result.total_mark'),
       accessor: 'final_grade',
-      Cell: ({value}) => value ? value + ' / ' + this.props.max_mark : '',
+      Cell: ({value}) => value + ' / ' + this.props.max_mark,
```

Comment:
The string concatenation of `undefined` with `' / ' + this.props.max_mark` would lead to something like `"undefined / 10"`, wouldn't it? This isn't great. We could do something like `(value === undefined ? '-' : value) + ' / ' + ...`